native/service/governance/neo3_state_manager/param.go

```diff
@@ -0,0 +1,75 @@
+package neo3_state_manager
+
+import (
+	"fmt"
+	"github.com/polynetwork/poly/common"
+)
+
+type StateValidatorListParam struct {
+	StateValidators []string       // public key strings in encoded format, each is 33 bytes in []byte
+	Address         common.Address // for check witness?
```

Comment:
which place will use `Address`?

Reply:
I am not sure, maybe when calling the contract methods. `RelayerListParam` uses `Address`:
```go
type RelayerListParam struct {
	AddressList []common.Address
	Address     common.Address
}
```